test/controllers/actions/FactCheckerAuthenticatedIdentifierActionSpec.scala

```diff
@@ -33,7 +33,7 @@ import scala.concurrent.{ExecutionContext, Future}
 class FactCheckerAuthenticatedIdentifierActionSpec extends ControllerBaseSpec with MockAuthConnector {
```

Comment:
Same as above if we changed the main controller name then perhaps we could have changed the test Spec name ?

Reply:
As above

Reply:
Found a few more in same package, updating